src/style.css

```diff
@@ -0,0 +1,82 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  font-size: 100px;
+  color: white;
+  font-family: Arial, Helvetica, sans-serif;
+  flex-direction: column;
+  align-items: flex-start;
```

Comment:
you don't have display flex on this class, this won't work and it's actually redundant
```suggestion
```